src/components/custom-field-input-single-choice/custom-field-input-single-choice.jsx

```diff
@@ -94,8 +130,8 @@ const ChoiceType = PropTypes.shape({
 });
 
 CustomFieldInputSingleChoice.propTypes = {
+  /** Needs to be the Custom Field ID from the database for self-loading of choices */
```

Comment:
I just realized this might not be the case in our builder.

We have two instance of a particular single choice:

1. Canvas
2. Popover default value

Both of these are single choice fields that need to load their own data.

Unfortunately, we use `id` in our React components as a HTML `id` attribute -- where we consider it to be _unique_ on the DOM. 

We will want to use a new prop for the custom field id. I'm okay with `fieldId` or `customFieldId`. You can come up with something too.